modules/indexer/code/bleve.go

```diff
@@ -322,20 +338,26 @@ func nonGenesisChanges(repo *models.Repository, revision string) (*repoChanges,
 
 func processRepoIndexerOperationQueue() {
 	for {
-		op := <-repoIndexerOperationQueue
-		var err error
-		if op.deleted {
-			if err = deleteRepoFromIndexer(op.repoID); err != nil {
-				log.Error("deleteRepoFromIndexer: %v", err)
+		select {
+		case op := <-repoIndexerOperationQueue:
```

Comment:
In reality this queue is not persistent and will be lost on shutdown.

Reply:
Agreed.